CHANGELOG.md

```diff
@@ -2088,3 +2088,4 @@ This release has several exciting features, the most notable of them being setti
 * [FEATURE] You can specify "heap ballast" to reduce Go GC Churn #1489
 * [BUGFIX] HA Tracker no longer always makes a request to Consul/Etcd when a request is not from the active replica #1516
 * [BUGFIX] Queries are now correctly cancelled by the query-frontend #1508
+* [ENHANCEMENT] AlertManager: Retrying AlertManager Get Requests (Get Alertmanager status, Get Alertmanager Receivers) on next replica on error #4840
```

Comment:
New changelog entry should be on top, notunder the 0.1.0 release.

Reply:
updated the changelog entry and placed this change under  ## master / unreleased